packages/volto/src/components/manage/Widgets/ImageWidget.jsx

```diff
@@ -60,6 +62,15 @@ const messages = defineMessages({
     id: 'Uploading image',
     defaultMessage: 'Uploading image',
   },
+  imageUploadErrorTitle: {
+    id: 'imageUploadErrorTitle',
+    defaultMessage: 'Error',
+  },
+  imageUploadErrorMessage: {
+    id: 'imageUploadErrorMessage',
+    defaultMessage:
+      'An error occured while uploading the image, please try again.',
```

Comment:
```suggestion
      'Please upload an image instead.',
```
The error is that the user uploaded something that's not an image. They are requested to upload an image.